protos/guard/guard.proto

```diff
@@ -0,0 +1,168 @@
+syntax = "proto3";
```

Comment:
do you want keep this gurad.proto in this PR, is there any difference between merged one and this one.
Or maybe remove in this PR.

Reply:
It is the original service, as Guard need to provide rpc StatusInfo , so it should be merged into master with this PR.